substrate/frame/support/procedural/src/benchmark.rs

```diff
@@ -723,6 +724,10 @@ fn expand_benchmark(
 		Ok(ident) => ident,
 		Err(err) => return err.to_compile_error().into(),
 	};
+	let frame_system = match generate_crate_access_2018("frame-system") {
+		Ok(path) => path,
+		Err(err) => return err.to_compile_error().into(),
+	};
```

Comment:
Curious why we didn't need to match statement above

Reply:
The previous fn returns a Result, but this one doesn't